common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -2,22 +2,20 @@
     <div class="indicator_container">
     % if input_type == 'checkbox' or not value:
         <span class="status ${status.classname if show_correctness != 'never' else 'unanswered'}"
-            id="status_${id}"
-            aria-describedby="inputtype_${id}">
+            id="status_${id}">
             <span class="sr">
                 %for choice_id, choice_description in choices:
                 % if choice_id in value:
                     ${choice_description}, 
                 %endif
                 %endfor
-                -
                 ${status.display_name}
             </span>
         </span>
     % endif
     </div>
 
-    <fieldset role="${input_type}group" aria-label="${label}">
+    <fieldset aria-label="${label}">
```

Comment:
@ahsan-ul-haq we won't need an ```aria-label``` here, so you can remove it.  The accessible label comes from the ```<legend>``` element below